canonical-web-and-design/maas-ui
# Display a table of dhcp snippets

Done:
- Populated the DHCP table with data from MAAS. Fixes: #74.

QA:
- Vist /dhcp
- Check that you see dhcp data and that you can expand rows, search etc.
